src/components/RecommendedFile.vue

```diff
@@ -11,7 +11,9 @@
 		@click.prevent="navigate"
 		@keyup.enter.prevent="navigate">
 		<div class="thumbnail"
-			:style="{ 'background-image': 'url(' + previewUrl + ')' }" />
+			:style="{ 'background-image': isFolder ? '' : 'url(' + previewUrl + ')' }">
+			<FolderIcon v-if="isFolder" />
+		</div>
```

Comment:
Does this not work? Might get rid of the `:deep` then.
```vue
<FolderIcon v-if="isFolder" class="thumbnail" />
<div v-else class="thumbnail" :style="{ 'background-image': 'url(' + previewUrl + ')' }" />
```

Reply:
Works too sure :+1:

Reply:
Done @susnux !!